src/plugins/authentication/firebase/firebaseInit.js

```diff
@@ -0,0 +1,5 @@
+import firebase from 'firebase'
```

Comment:
This is bad .... i see this is being used in main and it should be Init.js not firebase specific. so that each plugin can name it as init lifecycle method rather than cognito implementing firebaseInit.js